### PR TITLE
[DO NOT MERGE] Adding logic to the constructor for wellness encounters. 

### PR DESCRIPTION
**Pull request by adamdavis40208**
Wellness encounters will be only on weekdays, during business hours.

### CODE REVIEW

File: src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java [hunk 1]
```diff
@@ -310,6 +311,20 @@ public BigDecimal total() {
 
     public Encounter(long time, String type) {
       super(time, type);
+      if (type.equalsIgnoreCase(EncounterType.WELLNESS.toString())){
```
Comment (adamdavis40208) [inline src/main/java/org/mitre/synthea/world/concepts/HealthRecord.java:314]: The meat of this is here, the constructor detects if the type is WELLNESS, and then verifies that the days are weekdays and the times are business hours

File: src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    @Test
+    public void testEncounterWellness() {
+        Encounter encounter = person.record.encounterStart(time,
+                HealthRecord.EncounterType.WELLNESS.toString());
+        assertEquals("WELLNESS", encounter.type);
+        long timeArray[] = new long[]{ encounter.start, encounter.stop };
+        for(long l: timeArray){
+            Calendar c = Calendar.getInstance();
+            c.setTimeInMillis(l);
+            // Insure start/stop are a weekday
```
Comment (stpapp) [inline src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java]: Nit:  Insure -> Ensure

File: src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            assertEquals(c.get(Calendar.DAY_OF_WEEK) != Calendar.SATURDAY, true);
+            assertEquals(c.get(Calendar.DAY_OF_WEEK) != Calendar.SUNDAY, true);
+            // Insure start/stop are during business hours
```
Comment (stpapp) [inline src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java]: Nit: Insure -> Ensure

File: src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    public void testEncounterWellness() {
+        Encounter encounter = person.record.encounterStart(time,
+                HealthRecord.EncounterType.WELLNESS.toString());
+        assertEquals("WELLNESS", encounter.type);
+        long timeArray[] = new long[]{ encounter.start, encounter.stop };
+        for(long l: timeArray){
+            Calendar c = Calendar.getInstance();
+            c.setTimeInMillis(l);
+            // Insure start/stop are a weekday
+            assertEquals(c.get(Calendar.DAY_OF_WEEK) != Calendar.SATURDAY, true);
```
Comment (stpapp) [inline src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java:50]: Will this fail if the test is run on a Saturday?

File: src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java (hunk 3 above)
Comment (adamdavis40208) [reply src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java]: holy shit what is wrong with my brain

File: src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java (hunk 4 above)
Comment (adamdavis40208) [reply src/test/java/org/mitre/synthea/modules/EncounterModuleTest.java:50]: Well, that might be a nice thing to test. Let me hardcode a date into the person that'll set a weekend.

**Pull request closed without merging**